.github/workflows/ansible-managed-var-comment.yml

```diff
@@ -0,0 +1,26 @@
+---
+name: Check for ansible_managed variable use in comments
+on:  # yamllint disable-line rule:truthy
+  pull_request:
+  push:
+    branches:
+      - main
+      - master
+  workflow_dispatch:
+jobs:
+  ansible_managed_var_comment:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout repo
+        uses: actions/checkout@v3
+      - name: Install tox, tox-lsr
+        run: |
+          set -euxo pipefail
+          python -m pip install --upgrade pip
+          sudo apt-get update
+          sudo apt-get install -y git
+          pip install "git+https://github.com/linux-system-roles/tox-lsr@2.13.1"
```

Comment:
i wonder whether this would be better available in some requirements file, so it is centralized, and we can add dependabot to update it automatically

(this applies also to all the other actions added in this PR)

Reply:
see https://github.com/linux-system-roles/.github/pull/7